src/components/FolderCreationModal.jsx

```diff
@@ -30,6 +30,7 @@ const FolderCreationModal = ({
   siteName,
   setIsFolderCreationActive,
 }) => {
+  console.log(pagesData)
```

Comment:
nit: remove log statement

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/412/commits/f54ce6788955e338947448a8ef1423e7cb07e919